configs/scripts/program/build.sh

```
@@ -1,125 +1,38 @@
 #!/bin/bash
 
-set -euo pipefail
-
 SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 OUTPUT="./programs/.bin"
+# saves external programs binaries to the output directory
+source ${SCRIPT_DIR}/dump.sh ${OUTPUT}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)